.github/workflows/tests.yml

```diff
@@ -832,10 +838,9 @@ jobs:
               # to override the version with a derivative of the branch name
 
               # override the version if `git describe --tag` does not start with the branch version
-              last_release = check_output(["git", "describe", "--tag"], text=True).strip()
-              prefix = "${{ github.ref_name }}"[:-1]  # without x suffix
-              if not last_release.startswith(prefix):
-                  envs["VERSION_OVERRIDE"] = "${{ github.ref_name }}"
+              last_tag = check_output(["git", "describe", "--tag", "--abbrev=0"], text=True).strip()
+              num_commits = check_output(["git", "rev-list", "--count", "HEAD", f"{last_tag}..HEAD"], text=True).strip()
+              envs["VERSION_OVERRIDE"] = f"{last_tag}rc{num_commits}"
```

Comment:
I don't think this addresses the issue outlined in the comment above

Reply:
@kenodegard,

I tested this manually by adding a new tag and seeing what version would be generated and it seemed to work. This unfortunately super difficult to test.

I would prefer to just fix this as on the release branch itself if there are any problems because at least we'll be able to test it directly there.

Reply:
The special casing (`if not last_release.startswith(prefix)`) was to ensure that when a release branch was first created (but before a release had been made with a tag) we would derive the version from the branch name and not from the last tag

Reply:
But why don't we just always tag the branch first then?

Reply:
with what tag? some `rc0` tag? yes I considered that before, but opted not to since thats another manual step in the release process

Reply:
Okay, I'll put that logic back in there.